src/settings.rs

```diff
@@ -225,6 +227,8 @@ pub(crate) fn metering_set_bytecode_const() -> u64 {
     METERING.set_bytecode_const()
 }
 
+pub(crate) fn metering_initial_const() -> u64 { METERING.initial_const() }
```

Comment:
I don't see this function being used besides in the test. I reckon it should be subbed to the `run_main` `limit` parameter right after the call. Also if you have questions or need precisions on the runtime module don't hesitate to ask we can have a call 😄